src/app/layout/main/main.component.ts

```diff
@@ -9,21 +9,7 @@ import { ArticleModel } from '../../models';
     styleUrls: ['./main.component.less'],
 })
 export class MainComponent implements OnInit {
-    posts = [
-        {
-            title: 'test',
-            description: 'asasa',
-            likesCount: 2,
-            username: 'Fattoh',
-            date: 'November 16, 2021',
-        },
-        {
-            title: 'test2',
-            description: 'new post 2',
-            username: 'Anton',
-        },
-    ];
-    tags$: Observable<{ tags: Array<string> }> = of({tags: []});
+    tags$: Observable<{ tags: Array<string> }>;
```

Comment:
```suggestion
    tags$: Observable<{ tags: string[] }>;
```

to be consistent with `posts$: Observable<{ articles: ArticleModel[] }>;`